### PR TITLE
propositions to tend to a working version on OSX

### PR DESCRIPTION
**Pull request by luclaurent**
hi Benoit,
New propositions
be careful, I used Python 3.x and tikz seems to not work anymore with latexml

…script + use Pillow and pdf2image (pip install) in place of pythonmagick + new test LaTeX's test case with tikz), works with Python 3.7.6 , bug with generated xml (fraction field engenders errors but import will work) + tikz (texlive 2020) is not working with latexml 0.8.4

### CODE REVIEW

File: src/grading.py [hunk 1]
```diff
@@ -41,12 +41,51 @@
...
+        if fileIn is not None and fileOut is not None:
+            self.convertImage(fileIn,fileOut)
+
+    def loadImageTool(self):
+        """
+        chargement pythonmagick si disponible
+        """
+        loadok = False
+        try:
+            global Image
```
Comment (nennigb) [inline src/grading.py:62]: we could avoid global here

File: src/grading.py [hunk 2]
```diff
@@ -419,9 +460,11 @@ def EncodeImg(Ii,pathin,pathout):
...
 ############################################################################
 # écriture fichier out
-xmlout =  open(fileout, 'w')
-result_tree.write(xmlout, pretty_print=True,encoding="utf-8")
-xmlout.close()
+# xmlout =  open(fileout, 'w')
+# result_tree.write(xmlout, pretty_print=True,encoding="utf-8")
+# xmlout.close()
+result_tree.write(fileout, pretty_print=True,encoding="utf-8")
+print(result_tree)
```
Comment (nennigb) [inline src/grading.py:467]: remove the print for production

File: amc2moodle.sh [hunk 3]
```diff
@@ -88,17 +88,17 @@ EOF
 # function to init the requiered variable
 function init() {
     # get absolute name of the input file
-    filein=$(readlink -m $OPTARG) # do not check existance!
+    filein=$(realpath $OPTARG) # do not check existance!
```
Comment (nennigb) [inline amc2moodle.sh:91]: Could you also replace the other readlink

File: src/grading.py (hunk 1 above)
Comment (luclaurent) [reply src/grading.py:62]: I am not sure... I had an error without the global declaration

File: src/grading.py (hunk 2 above)
Comment (luclaurent) [reply src/grading.py:474]: Done and other print (use deb option for some of it)

File: amc2moodle.sh (hunk 3 above)
Comment (luclaurent) [reply amc2moodle.sh:91]: done but not necessary (see new proposition with Python script)

**Comment by luclaurent** [comment]
many new propositions:
* a new script (amc2moodle which is based on the amc2moodle.py script) accessible from any folder
* use Wand python module (but Pythonmagick, and Pillow/pdf2image remain available)
* a new class to manage the building process
* many small fixes in gradin.py
* setuptools is now used to install the script (using pip install . or python setup.py install)
* update of the building of the documentation
* update of the documentation it self

Some files could be removed now.
Enjoy :-)

File: src/grading.py (hunk 1 above)
Comment (luclaurent) [reply src/grading.py:62]: I don't know but I had an error without it

File: src/grading.py (hunk 2 above)
Comment (luclaurent) [reply src/grading.py:474]: done

File: amc2moodle.sh (hunk 3 above)
Comment (luclaurent) [reply amc2moodle.sh:91]: done but not necessary from now (see my new script proposition)

**Comment by luclaurent** [comment]
update README

**Comment by luclaurent** [comment]
new syntax for importlib (adpated for github actions) + add missing file

**Pull request merged**